learn/parse_http.go

```diff
@@ -311,28 +308,7 @@ func parseBody(contentType string, bodyStream io.Reader, statusCode int) (*pb.Da
 	// TODO: XML parsing
 	// TODO: application/json-seq (RFC 7466)?
 	// TODO: more text/* types
-	var parseBodyDataAs pb.HTTPBody_ContentType
-	switch mediaType {
```

Comment:
Please capture these TODOs next to the list of content-types, rather than leaving them stranded here?